html/client/index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE HTML>
+<html ng-app="index">
+    <head>
+        <title>ConnectBox</title>
```

Comment:
The changed title is causing a test failure

```
$ TEST_IP=172.28.128.3 python -m unittest discover tests
.....F
======================================================================
FAIL: testPageTitle (test_biblebox_static.BibleBoxWebDriverTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/esteele/Code/biblebox-pi/tests/test_biblebox_static.py", line 65, in testPageTitle
    self.assertIn("The BibleBox", self.browser.title)
AssertionError: 'The BibleBox' not found in u'ConnectBox'

----------------------------------------------------------------------
```

I don't mind if you update the title, or update the test.